### ISSUE TITLE
Controlling recording status from MQTT

### ISSUE THREAD

**Issue by MatteoGheza** [QuantumEntangledAndy/neolink#313]
**Is your feature request related to a problem? Please describe.**
I’d like to control the continuous recording status of my Reolink camera through MQTT actions, similar to other control functions. Currently, no dedicated MQTT topic exists for toggling recording, which limits automation options.

**Describe the solution you'd like**

A new MQTT topic could be added to control the recording state directly, allowing users to start and stop recording as needed.

_Suggested Control Message_
/control/recording — Toggles recording status with payload options:
"start": Initiates continuous recording
"stop": Ends continuous recording
Suggested Status Message
/status/recording — Returns the current recording status ("on" or "off") in response to the recording toggle or as part of regular status updates.

Adding this control and status feature would allow seamless integration of recording functionality into MQTT-based home automation systems, enabling actions like starting recording based on motion sensors or other triggers.

**Comment by QuantumEntangledAndy** [comment]
I take it you mean recording onto the SD card? A lot of the SD card functions haven't been reverse engineered fully yet so this request might take awhile

**Comment by hblanken** [comment]
Yes, this would be fantastic - controlling SD-card recordings and pulling recorded media from SD-card.

**Comment by smurfy** [comment]
> I take it you mean recording onto the SD card? A lot of the SD card functions haven't been reverse engineered fully yet so this request might take awhile

Something like this would be really useful, any tipp how i would start RE it?
